src/Playground.jl

```diff
@@ -18,92 +20,71 @@ include("clean.jl")
 
 
 export
-    # methods
-    main,
-    argparse,
-    load_config,
-    install,
-    dirinstall,
-    #gitinstall,
-    create,
-    activate,
-    execute,
-    list,
-    clean,
-    config_path,
+    # # methods
```

Comment:
I need to update the export list. Some of the functions changed names which is why I originally just commented out the export list.